README.markdown

```diff
@@ -157,6 +157,28 @@ To use a mirror of the iojs binaries, set `$NVM_IOJS_ORG_MIRROR`:
 
 `nvm use` will not, by default, create a "current" symlink. Set `$NVM_SYMLINK_CURRENT` to "true" to enable this behavior, which is sometimes useful for IDEs.
 
+### Deeper Shell Integration
+
+Those options are all optional and not supported. We are accepting pull requests for more examples.
+
+
+#### ZSH
```

Comment:
both `zsh` and `Zsh` seem to be common. But then again one can upcase lowercase names in headlines, can one?